custom-targets/util/clouddeploy/clouddeploy.go

```diff
@@ -492,3 +493,30 @@ func parseGCSURI(uri string) (gcsObjectURI, error) {
 	}
 	return obj, nil
 }
+
+// isDeployParamAndKey determines if the provided env var key corresponds
+// to a deploy parameter, if it is then it returns the deploy parameter key.
+func isDeployParamAndKey(key string) (bool, string) {
+	if strings.HasPrefix(key, cloudDeployCustomTargetPrefix) {
+		transformedKey := strings.TrimPrefix(key, cloudDeployCustomTargetPrefix)
+		transformedKey = fmt.Sprintf("customTarget/%s", transformedKey)
+		return true, transformedKey
+	} else if strings.HasPrefix(key, cloudDeployEnvVarPrefix) {
+		return false, ""
+	} else {
+		return true, key
+	}
+}
+
+// FetchDeployParameters returns a  map of all the deploy parameters provided in the execution environment
```

Comment:
nit: end comment with a period. Also you have two spaces in between "a" and "map"